arch/i386/common.S

```diff
@@ -0,0 +1,6 @@
+#include "utils/asm.h"
+
+GLOBAL(get_pc_thunk)
```

Comment:
It should be `ENTRY` instead of `GLOBAL`.  The difference is the visibility of the symbol - a `GLOBAL` symbol will be visible from outside of libmcount while `ENTRY` symbol will be visible only in libmcount.

Reply:
@namhyung ok, I see!